sys-devel/gcc/files/gcc-12.1.0-avoid-null-terminated-name-collision-with-macos-13-sdk.patch

```diff
@@ -0,0 +1,81 @@
+From d1201dbf55a11d391030914985ba6b443e59baa5 Mon Sep 17 00:00:00 2001
```

Comment:
Please include a link to the upstream commit and any other useful references at the top of the patch.